src/DataStructures/Tensor/Expressions/Product.hpp

```diff
@@ -190,3 +190,18 @@ SPECTRE_ALWAYS_INLINE auto operator*(
   return TensorExpressions::NumberAsExpression(number) * t;
 }
 /// @}
+
+/// \ingroup TensorExpressionsGroup
+/// \brief Returns the tensor expression representing the negation of a tensor
+/// expression
+///
+/// \note The implementation uses the operation, `-1.0 * t`
+///
+/// \param t the tensor expression
+/// \return the expression representing the negation of the tensor expression
+template <typename T>
+SPECTRE_ALWAYS_INLINE auto operator-(
+    const TensorExpression<T, typename T::type, typename T::symmetry,
+                           typename T::index_list, typename T::args_list>& t) {
+  return TensorExpressions::NumberAsExpression(-1.0) * t;
```

Comment:
I think you want to actually do a unary minus on the underlying type. It's generally not a good idea to stack expressions subtly like this. I'm worried it'll have performance consequences. Can you write a TE class that has a get return `-t_.get();`?